db/migrate/20201215010551_create_market_places.rb

```diff
@@ -0,0 +1,17 @@
+class CreateMarketPlaces < ActiveRecord::Migration[6.0]
+  def change
+    create_table :market_places do |t|
```

Comment:
If you want to add the reference with the user table, you can use the following command on migrate:

` t.belongs_to :user, null: false, foreign_key: true`

Also in the MarketPlace model you can place the reference:

`belongs_to: user`